examples/expressions.rs

```diff
@@ -57,6 +57,12 @@ impl Checker {
   }
 }
 
+#[cfg(not(feature = "tensorflow_unstable"))]
+fn run() -> Result<(), Box<Error>> {
+  Ok(())
```

Comment:
Please print out a message explaining that the example is disabled because the tensorflow_unstable feature is not enabled.